01_pipeline/03_resources.yaml

```diff
@@ -7,7 +7,7 @@ spec:
   type: git
   params:
   - name: url
-    value: http://github.com/openshift-pipelines/vote-api.git
+    value: http://github.com/paolocarta/vote-api.git
```

Comment:
Is this change intentional?

Reply:
Sorry, please ignore it. Got it messed up with my local experiments.